ftw/news/browser/templates/news_listing_block.pt

```diff
@@ -14,7 +14,7 @@
            i18n:translate="">No content available</p>
 
         <ul class="news-row">
-            <li class="news-item" tal:repeat="item news">
```

Comment:
The `news-item` class is both on the `li` and `a` tag. On one hand this is confusing and on the other hand the styling for `news-item` is applied to both tags which leads to strange behaviors such as duplicated borders or margins.